app/utils/translation.py

```diff
@@ -15,7 +15,7 @@ def load_translation(language):
     primary_language = language.split(",")[0].split(";")[0].split("-")[0]
```

Comment:
Here comes a copy of my comment about loading data with I/O. Here it is the case as it is made everytime as it depends on the language. However, it is possible to cache it. You may use the lru_cache decorator or you may use a dictionary and override `__getattr__` to lazy_load the values on the first call.

What I wrote lately: decorate the function with @lru_cache https://docs.python.org/3/library/functools.html#functools.lru_cache use that function from main.py it will speed up the app on subsequent calls, even more with a growing *.json file and even more on the server

Reply:
Implemented, in the simplest version.